<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomPaymentDialog</class>
 <widget class="QDialog" name="CustomPaymentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zaawansowane płatności</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxPayment1">
     <property name="title">
      <string>Płatność 1:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="hBoxLayoutPayment1">
        <item>
         <layout class="QHBoxLayout" name="hBoxLayoutPayment1_2">
          <item>
           <widget class="QLabel" name="labelPayment1">
            <property name="text">
             <string>Płatność:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>comboBoxPayment1</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxPayment1"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hBoxLayoutDate1">
          <item>
           <widget class="QLabel" name="labelDayOfPayment1">
            <property name="text">
             <string>Termin:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>dateEditDayOfPayment1</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEditDayOfPayment1">
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="hBoxLayoutAmount1">
        <item>
         <widget class="QLabel" name="labelAmount1">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Kwota:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBoxAmount1</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxAmount1">
          <property name="maximum">
           <double>99999.990000000005239</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxPayment2">
     <property name="title">
      <string>Płatność 2:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="hBoxLayoutPaymentLower">
        <item>
         <layout class="QHBoxLayout" name="hBoxLayoutPayment2">
          <item>
           <widget class="QLabel" name="labelPayment2">
            <property name="text">
             <string>Płatność:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>comboBoxPayment2</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxPayment2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hBoxLayoutDate2">
          <item>
           <widget class="QLabel" name="labelDayOfPayment2">
            <property name="text">
             <string>Termin:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>dateEditDayOfPayment2</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEditDayOfPayment2">
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="hBoxLayoutAmount2">
        <item>
         <widget class="QLabel" name="labelAmount2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Kwota:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBoxAmount2</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxAmount2">
          <property name="maximum">
           <double>99999.990000000005239</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hBoxLayoutButtons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>&amp;Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxPayment1</tabstop>
  <tabstop>dateEditDayOfPayment1</tabstop>
  <tabstop>doubleSpinBoxAmount1</tabstop>
  <tabstop>comboBoxPayment2</tabstop>
  <tabstop>dateEditDayOfPayment2</tabstop>
  <tabstop>doubleSpinBoxAmount2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
